### PR TITLE
feat: created the master readme for the repo

### PR DESCRIPTION
**Pull request by Shashwat79802**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+<p align="center">
+
+  <a href="CODE_OF_CONDUCT.md" alt="Contributions welcome">
+    <img src="https://img.shields.io/badge/Keploy-Rust-orange" /></a>
+
+  <a href="CODE_OF_CONDUCT.md" alt="Contributions welcome">
+    <img src="https://img.shields.io/badge/Contributions-Welcome-brightgreen?logo=github" /></a>
+
+  <a href="https://join.slack.com/t/keploy/shared_invite/zt-12rfbvc01-o54cOG0X1G6eVJTuI_orSA" alt="Slack">
+    <img src=".github/slack.svg" /></a>
```
Comment (Sonichigo) [inline README.md:11]: .github folder doesn't exist so images are breaking

**Review by Sonichigo — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
...
+
+  <a href="https://opensource.org/licenses/Apache-2.0" alt="License">
+    <img src=".github/License-Apache_2.0-blue.svg" /></a>
+
+  <a href="https://opensource.org/licenses/Apache-2.0" alt="License">
+    <img src=".github/License-Apache_2.0-blue.svg" /></a>
```
Comment (Sonichigo) [inline README.md]: licence is present twice! And images are still breaking please check once!

**Review by Sonichigo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Shashwat79802) [reply README.md]: image
The images aren't breaking locally. Fixed the license part!!

**Review by Sonichigo — APPROVED** [review]
LGTM

**Pull request merged**